package.json

```diff
@@ -40,7 +40,7 @@
     "datatables.net-bs": "^1.10.13",
     "datatables.net-dt": "^1.10.15",
     "elasticsearch": "13.0.0",
-    "font-awesome": "^4.7.0",
+    "font-awesome": "https://registry.npmjs.org/font-awesome/-/font-awesome-4.7.0.tgz",
```

Comment:
A https version is a bit odd, maybe a comment in the readme?

Reply:
Weird I think this might have been done automatically, I'll revert it back to 4.7.0